cpp/include/cudf/detail/aggregation/aggregation.hpp

```diff
@@ -367,7 +377,10 @@ class var_aggregation final : public std_var_aggregation {
  */
 class std_aggregation final : public std_var_aggregation {
  public:
-  std_aggregation(size_type ddof) : std_var_aggregation{aggregation::STD, ddof} {}
+  std_aggregation(size_type ddof)
+    : aggregation{aggregation::STD}, std_var_aggregation{aggregation::STD, ddof}
```

Comment:
```suggestion
    : std_var_aggregation{aggregation::STD, ddof}
```
Same for `var_aggregation` ctor as mentioned https://github.com/rapidsai/cudf/pull/8906/files#r680051532